### DISCUSSION TITLE
`librechat.yaml` file cached forever

### DISCUSSION THREAD

**Discussion by Lazyshot** [danny-avila/LibreChat#4080] [Troubleshooting]
### What happened?

When using redis for caching purposes the `librechat.yaml` file does not respect file edits, because the contents are loaded into the cache once and never invalidated.

The `getCustomConfig` code executes on startup and finds the alterations made and even logs out the new file, but when calling the `/api/config` endpoint no changes are observed. 

### Steps to Reproduce

1. Write `librechat.yaml` file
2. Enable Redis caching
3. Alter `librechat.yaml` file (e.g. model specs)
4. No Changes found on `/api/config`

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
This is the intended usage with redis right now, which is experimental. You will need to flush the cache manually

**Reply by Lazyshot** [reply]
As of right now, that is exactly what we do. We have a deploy step that flushes redis. 

Our primary use case for Redis is primarily the social media login session code synchronization.
